website/sidebars.json

```diff
@@ -28,27 +28,21 @@
       "external-configuration-parameter-values",
       {
         "type": "subcategory",
-        "label": "Cloud products",
+        "label": "Native product types",
```

Comment:
Native che informazione dovrebbe aggiungere? esistono product types non nativi che vanno citati in una sezione che si chiama products onboarding?

Reply:
hai ragione, mi è venuta un'idea... ora lo faccio